lms/static/sass/shared-v2/_modal.scss

```diff
@@ -26,12 +26,12 @@
     z-index: z-index(mid-front);
     background: $modal-bg-color;
     border-radius: 0;
-    border: 1px solid rgba(0, 0, 0, 0.9);
+    border: 1px solid $lms-border-color;
     box-shadow: inset 0 1px 0 0 rgba(255, 255, 255, 0.7);
     overflow: hidden;
-    padding-left: ($baseline/2) + px;
-    padding-right: ($baseline/2) + px;
-    padding-bottom: ($baseline/2) + px;
+    padding-left: ($baseline/2);
```

Comment:
Can these be simplified into a single property shorthand? Also keep in mind RTL.